[CI] Failures on 6.x for 60_ml_config_migration/Test old cluster jobs and datafeeds and delete them

This has failed at least twice since #36698 was merged to 6.x

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/693/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+matrix-java-periodic/ES_BUILD_JAVA=java11,ES_RUNTIME_JAVA=java11,nodes=virtual&&linux/127/console

```
./gradlew :x-pack:qa:rolling-upgrade:with-system-key:v5.6.15#upgradedClusterTestRunner -Dtests.seed=7721FEBE7CCB10B7 -Dtests.class=org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT -Dtests.method="test {p0=upgraded_cluster/60_ml_config_migration/Test old cluster jobs and datafeeds and delete them}" -Dtests.security.manager=true -Dtests.locale=ar-AE -Dtests.timezone=Etc/GMT-10 -Dcompiler.java=11 -Druntime.java=8 -Dtests.rest.suite=upgraded_cluster -Dtests.rest.blacklist="/30_ml_jobs_crud/Test model memory limit is updated"
```

```
00:50:45    > Throwable #1: java.lang.AssertionError: Failure at [upgraded_cluster/60_ml_config_migration:43]: datafeeds.0.state didn't match expected value:
00:50:45    >              datafeeds.0.state: expected [started] but was [stopped]
00:50:45    > 	at __randomizedtesting.SeedInfo.seed([7721FEBE7CCB10B7:FF75C164D2377D4F]:0)
00:50:45    > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:406)
00:50:45    > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:383)
00:50:45    > 	at java.lang.Thread.run(Thread.java:748)
00:50:45    > Caused by: java.lang.AssertionError: datafeeds.0.state didn't match expected value:
00:50:45    >              datafeeds.0.state: expected [started] but was [stopped]
00:50:45    > 	at org.elasticsearch.test.rest.yaml.section.MatchAssertion.doAssert(MatchAssertion.java:87)
00:50:45    > 	at org.elasticsearch.test.rest.yaml.section.Assertion.execute(Assertion.java:76)
00:50:45    > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:399)
00:50:45    > 	... 38 more
```

I get failures locally as well but they have a different assertion failure.

I'm going to mute this on 6.x

--- a/x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/upgraded_cluster/60_ml_config_migration.yml
+++ b/x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/upgraded_cluster/60_ml_config_migration.yml
@@ -8,9 +8,6 @@ setup:
 
 ---
 "Test old cluster jobs and datafeeds and delete them":
-  - skip:
-      version: "all"
-      reason: "@AwaitsFix: https://github.com/elastic/elasticsearch/issues/36810"
 
   - do:
       xpack.ml.get_jobs:

